lib/views/scan_view.dart

```diff
@@ -1,12 +1,46 @@
 import 'package:cycle_lock/controllers/scan_controller.dart';
 import 'package:flutter/material.dart';
 import 'package:get/get_state_manager/get_state_manager.dart';
+import 'package:qr_code_scanner/qr_code_scanner.dart';
 
 class ScanView extends GetView<ScanningController> {
   const ScanView({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+        appBar: AppBar(title: const Text('Cycle lock')),
+        body: Stack(alignment: Alignment.center, children: <Widget>[
+          Positioned(child: qrView(context)),
+          Positioned(bottom: 30, child: controlButtons())
+        ]));
   }
+
+  Widget qrView(context) => QRView(
+        key: controller.qrKey,
+        onQRViewCreated: controller.onQRViewCreated,
+        overlay: QrScannerOverlayShape(
+          borderColor: Colors.blue.shade200,
+          borderRadius: 10,
+          borderLength: 20,
+          borderWidth: 10,
+          cutOutSize: MediaQuery.of(context).size.width * 0.6,
+        ),
+      );
+
+  Widget controlButtons() => Container(
+        padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 5),
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(10),
+            color: Colors.blue.shade200),
+        child: Row(
+          mainAxisSize: MainAxisSize.max,
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: <Widget>[
+            IconButton(
+                onPressed: controller.flash, icon: Icon(Icons.flash_on_rounded))
```

Comment:
```
Analyzing cycle-lock-app...                                     

   info • Prefer const with constant constructors • lib/views/scan_view.dart:42:52 • prefer_const_constructors
```
[Lint error](https://github.com/delta/cycle-lock-app/runs/4151682883?check_suite_focus=true) is happening here. Resolve it